worldedit-core/src/main/java/com/sk89q/worldedit/command/BrushCommands.java

```diff
@@ -992,15 +992,15 @@ public void command(
                     Expression radius,
             @Arg(desc = "Command to run")
                     List<String> input,
-            @Switch(name = 'p', desc = "Show any printed output")
-                    boolean print
+            @Switch(name = 'h', desc = "Hide any printed output")
+                    boolean hide
```

Comment:
Not having output from this brush is awkward, especially since it only runs the command once